modules/merge/lfs.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2019 The Gitea Authors.
```

Comment:
How about use `modules/pull` but not `modules/merge` so that we could move more pull related codes to this package.

Reply:
Well we could just rename this when that's done but if you prefer I can do that now.

A few of the pipelines used in the lfs code are reusable, e.g. in #7199 - so may be some of those should be moved to git (or migrated to go-git code over time.)